css/index.css

```diff
@@ -0,0 +1,165 @@
+body {
+  background-color: gray;
+  margin: 0px;
+  padding: 0px;
+  overflow: hidden;
+}
+
+span {
+  padding: 0;
+  margin: 0;
+  cursor: pointer;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.main-navbar-div {
+  background-color: #0f0f0f;
+  padding: 0px 20px;
+  height: 70px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: fixed;
+  width: 100%;
+  top: 0;
+}
+
+.navbar-left-content {
+  display: flex;
+  align-items: center;
+  gap: 15px;
+}
+
+.humber-icon {
+  height: 40px;
+  width: 40px;
+  border-radius: 50%;
+  background-color: #272727;
+  display: flex;
+  justify-content: center;
+}
+
+.youtube-icon {
+  height: 25px;
+  min-width: 70px;
+}
+
+.main-search-bar {
+  display: flex;
+  width: 40%;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.navbar-search-bar {
+  display: flex;
+  align-items: center;
+  background-color: #272727;
+  height: 40px;
+  width: 90%;
+  border-radius: 20px;
+}
+
+.input-field {
+  width: 90%;
```

Comment:
use 
calc(100% - 50px)
so that search icon width is fixed

Reply:
Done.